reclaim.c

```diff
@@ -668,7 +668,8 @@ GC_INNER void GC_start_reclaim(GC_bool report_if_found)
             void **lim = &(GC_obj_kinds[kind].ok_freelist[MAXOBJGRANULES+1]);
 
             for (fop = GC_obj_kinds[kind].ok_freelist;
-                 (word)fop < (word)lim; (*(word **)&fop)++) {
+                 (word)fop < (word)lim; fop++) {
+                 //(word)fop < (word)lim; (*(word **)&fop)++) {
```

Comment:
Commented code.

Reply:
resolved in [8aeb81e2](https://github.com/capablevms/bdwgc/pull/5/commits/8aeb81e2bdcc074e8def623f78c632e4bf666a4b)